include/odp/api/spec/packet_io.h

```diff
@@ -845,6 +855,16 @@ typedef struct odp_pktio_capability_t {
 	/** Maximum number of output queues */
 	unsigned int max_output_queues;
 
+	/** Minimum output queue size
+	 *
+	 *  Zero if configuring queue size is not supported. */
+	uint32_t min_output_queue_size;
+
+	/** Maximum output queue size
+	 *
+	 *  Zero if configuring queue size is not supported. */
+	uint32_t max_output_queue_size;
```

Comment:
It should be less than or equal to ODP_PKTOUT_MAX_QUEUES. Right? If so, You add comment to convey the same.

Reply:
I assume you are referencing to `max_output_queues` field couple lines above. I'll add a clarification.

Reply:
Fixed in V2.